scripts/scil_visualize_bundlesList_mosaic.py

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import argparse
+from distutils.version import LooseVersion
+import os
+import shutil
+
+from dipy.viz import actor, window
+import nibabel as nib
+
+if LooseVersion(nib.__version__) < LooseVersion('2.1.0'):
+    raise ImportError("Unable to import the Nibabel streamline API. "
+                      "Nibabel >= v2.1.0 is required")
+
+import numpy as np
+
+from scilpy.io.utils import (
+    add_overwrite_arg, assert_inputs_exist, assert_outputs_exists)
+
+from PIL import Image
+from PIL import ImageFont
+from PIL import ImageDraw
+
+def build_arg_parser():
+    parser = argparse.ArgumentParser(
+        description='Visualize bundles from a list of bundles. The script will output '
```

Comment:
Visualize bundles from a list of bundles (remove " of bundles")
 per each bundles of (replace "per" with "for")

Also I would prefer a name such as "scil_visualize_bundles_mosaic.py"

Reply:
Text and name changed